{"name": "test_create_order_with_invalid_ingredient", "status": "failed", "statusDetails": {"message": "AssertionError: assert 403 == 500\n +  where 403 = <Response [403]>.status_code", "trace": "self = <test_create_order.TestCreateOrder object at 0x1031e8350>\nauth_token = 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjY3NDdjZWRjOWVkMjgwMDAxYjUyNjdlNCIsImlhdCI6MTczMjc2MDY2NSwiZXhwIjoxNzMyNzYxODY1fQ.XufTlbu3kLk8N9IKPDF2eVXCd4M2JmMSzHeattpqorI'\n\n    @allure.feature(\"Создание заказа\")\n    @allure.story(\"Попытка создать заказ с невалидными ингредиентами\")\n    def test_create_order_with_invalid_ingredient(self, auth_token):\n        order_data = {\"ingredients\": [\"invalid_hash\"]}\n        headers = {\"Authorization\": f\"Bearer {auth_token}\"}\n        response = requests.post(f\"{BASE_URL}/orders\", json=order_data, headers=headers)\n>       assert response.status_code == 500\nE       assert 403 == 500\nE        +  where 403 = <Response [403]>.status_code\n\ntest_create_order.py:38: AssertionError"}, "start": 1732760665119, "stop": 1732760665398, "uuid": "67bfe182-1bce-4ab6-8dac-65dfa87a8371", "historyId": "9bf7d65d22acbc46bdc4ddb4293ba878", "testCaseId": "9bf7d65d22acbc46bdc4ddb4293ba878", "fullName": "tests.test_create_order.TestCreateOrder#test_create_order_with_invalid_ingredient", "labels": [{"name": "story", "value": "Попытка создать заказ с невалидными ингредиентами"}, {"name": "feature", "value": "Создание заказа"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_order"}, {"name": "subSuite", "value": "TestCreateOrder"}, {"name": "host", "value": "admins-MacBook-Pro.local"}, {"name": "thread", "value": "25725-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_order"}]}